package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/johnnyon-amzn/dashboards-desktop-poc#readme",
   "devDependencies": {
-    "electron": "^18.1.0"
+    "electron": "18.3.7"
```

Comment:
why not use `^18.3.7`

Reply:
> why not use `^18.3.7`

https://nvd.nist.gov/vuln/detail/CVE-2022-36077, "This issue has been patched in versions: 21.0.0-beta.1, 20.0.1, 19.0.11, and 18.3.7."